fix: Update Redis Allowed IPs documentation (#1206)
scaleway/docs-content: managed-databases/redis/how-to/manage-allowed-ip-addresses-redis.mdx

  - postgresql-and-mysql
---

> Allowed IPs restrict the IPs allowed to access to your Database Instance. The entry `0.0.0.0/0` enables any host to establish a connection.

Reviewer: ```suggestion
Allowed IPs restrict the IPs allowed to access your Database Instance. The entry `0.0.0.0/0` enables any host to establish a connection.
```

3. Click **Allowed IPs** tab. 
4. Click **Add IP**. The IP can be a single IP or an IP block.
    <Message type="note">
      The IP must be specified in CIDR format, i.e. `198.51.100.135/32` for a single IP or `198.51.100.0/24` for an IP block.
    </Message>
5. Enter a single IP address or a sub-network.
    <Message type="note">
      To restore initial settings and allow connections from all IPs, add the following IP block to the list: `0.0.0.0/0`.
    </Message>
    
<Message type="note">

>       When no IP is specified, your cluster is unreacheable from any host. 

Reviewer 2: ```suggestion
      When no IP is specified, your cluster cannot be reached by any host.
```
